WordPress/gutenberg
# [RNMobile] Change ripple effect n BottomSheet settings

## Description

Prevent the Slider in BottomSheet settings to have ripple effect after press on its title.

Please also refer to:
[Related gutenberg-mobile PR](https://github.com/wordpress-mobile/gutenberg-mobile/pull/2023)
[Related gutenberg-mobile issue](https://github.com/wordpress-mobile/gutenberg-mobile/issues/2022)

## How has this been tested?




1. Add Stepper block (it has Slider in settings to control its height)
2. Open settings to see Slider
3. Press on Slider title to see if it has ripple effect
4. Check if you can check Slider value

- check if a11y works after proposed changes

## Screenshots 

**AFTER:**
<img src="https://user-images.githubusercontent.com/21242757/76754687-d2e73a80-6782-11ea-89d6-30fb16163223.gif" width="300"/>

**BEFORE:**
<img src="https://user-images.githubusercontent.com/21242757/76754694-d5499480-6782-11ea-969e-4d637b0e3f20.gif" width="300"/>

## Types of changes




Refactor: prevent ripple effect on Slider cell in BottomSheet settings (iOS and Android)
Refactor: disable ripple effect for BottomSheet rows on iOS

## Checklist:
